ProfileService/MainModule/View/MainViewController.swift

```diff
@@ -45,16 +65,24 @@ private extension MainViewController {
     
 }
 
-// MARK: - Set
+// MARK: - Private Methods
```

Comment:
я говорил что такие марки не очень так как слишком общие
Здесь лучше назвать - Binding to ViewModel
```suggestion
// MARK: - Private Methods
```